lib/components/narrative/narrative-itineraries.js

```diff
@@ -435,13 +435,13 @@ class NarrativeItineraries extends Component {
                     </S.ModeResultContainer>
                   )
                 })
-              ) : (
+              ) : !pending ? (
```

Comment:
I think this is causing results not to "trickle" in. On slow searches I think this is a bit annoying? Or am I just seeing things

Reply:
Oh I see. When viewing one itinerary, however, not having that will flash multiple itineraries until `pending` is `false`, and even show a different itinerary than the one originally being saved.

Reply:
Reverting that change for this PR (https://github.com/opentripplanner/otp-react-redux/pull/916/commits/ac129ae71f6e174ff0a865cf682c7bc30714a26f).